main.go

```diff
@@ -0,0 +1,47 @@
+package main
+
+import (
+	"./migraph"
+	"html/template"
+	"net/http"
+)
+
+func indexHandler(w http.ResponseWriter, r *http.Request) {
+	t, err := template.ParseFiles("index.html")
```

Comment:
This is a bit troublesome as this requires the `index.html` file to be in the current running directory. Making the contents of the `index.html` file a string const stored in a separate `.go` file would make the program to be standalone.